### PR TITLE
editcomp: Fix broken table layout when one game has a long name

### PR DESCRIPTION
**Pull request by dfabulich**
Fixes https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/498

Before:
image

After: 
image

### CODE REVIEW

**Comment by salty-horse** [comment]
MDN [warns not to use `<nobr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr), which is deprecated, and suggests `style="white-space: nowrap;"` instead.

Will it work for these elemenets?

**Comment by dfabulich** [comment]
Replaced with `<span class='nobr'>` which does the same thing.

File: www/combobox.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
 function makeComboBox($name, $textlen, $curval, $vals, $onSet = "null")
 {
     // write the input field
-    $txt = "<input type=\"text\" name=\"$name\" id=\"$name\" "
+    $txt = "<span class='nobr'><input type=\"text\" name=\"$name\" id=\"$name\" "
```
Comment (salty-horse) [inline www/combobox.php:26]: Now there's a mix of `"` and `'` in the generated HTML, in case it matters to you.

**Review by salty-horse — APPROVED** [review]

File: www/combobox.php (hunk 1 above)
Comment (dfabulich) [reply www/combobox.php:26]: doesn't matter to me 🤷

**Pull request merged**